components/infobox/wikis/starcraft2/infobox_person_player.lua

```diff
@@ -0,0 +1,556 @@
+local Player = require('Module:Infobox/Person')
+local Variables = require('Module:Variables')
+local Achievements = require('Module:Achievements in infoboxes')._player
+local RaceIcon = require('Module:RaceIcon').getBigIcon
+local String = require('Module:StringUtils')
+local Matches = require('Module:Upcoming ongoing and recent matches player/new')
+
+local _EPT_SEASON = 2021
```

Comment:
When does one season change into the next? Is it possible to figure it out dynamically using today's yy-mm-dd?

Reply:
a season ends sometime after iem katowice and before the first esl open cup that gives points
it all depends on esl, so no option to automate it